package.json

```diff
@@ -62,6 +62,7 @@
     "webpack": "^3.4.1",
     "webpack-bundle-analyzer": ">=3.3.2",
     "webpack-dev-server": "^2.5.1",
-    "webpack-stats-plugin": "^0.1.5"
+    "webpack-stats-plugin": "^0.1.5",
+    "optimize-css-assets-webpack-plugin": "^3.2.0"
```

Comment:
Looks like prior to this the list was sorted. Mind keeping it that way? Also looks like this needs RPM packaging.

Reply:
sure, I'll keep the order :+1: 
it's a dev dependency. do we still need RPM packaging for it?

Reply:
My understanding is this is used when building assets, so required during rpm build

Reply:
Indeed. If NPM packaging had a difference between dev, test and runtime you could ignore test but the other two are still needed (for Foreman) since the build process runs webpack.

Reply:
@amirfefer could we split that to different PR? We can introduce the Global css and optimize the css assets separatly, right?